composer.json

```diff
@@ -204,40 +207,40 @@
         "cweagans/composer-patches": "^1.5.0",
         "composer/installers": "^1.0",
         "drupal-composer/drupal-scaffold": "^2.0.0",
-        "drupal/core": "~8.4.8",
-        "drupal/features": "3.5",
+        "drupal/core": "8.6.7",
```

Comment:
let's put ~8.6.7 to get support for newer versions

Reply:
Fixed